src/app/domain/geolocation/hooks/tracking.ts

```diff
@@ -60,18 +48,13 @@ export const useGeolocationTracking = (): void => {
     void initializeTracking()
   }, [client])
 
-  const webhookQuery = buildServiceWebhookQuery()
-  const webhookResp = useQuery(webhookQuery.definition, webhookQuery.options)
-
-  if (Array.isArray(webhookResp.data) && webhookResp.data.length > 0) {
-    const data = webhookResp.data as WebhookTrigger[]
+  useEffect(() => {
+    const initializeWebhook = async (): Promise<void> => {
+      if (!client) return
 
-    const openpathServiceWebHook = data.find(
-      trigger => trigger.message.name === FETCH_OPENPATH_TRIPS_SERVICE_NAME
-    )
-    const webhook = openpathServiceWebHook?.links?.webhook
-    if (webhook) {
-      void storeFetchServiceWebHook(webhook)
+      await fetchWebhook(client)
```

Comment:
The naming might be a bit misleading, as the primary goal of the function is to store the webhook value in local storage.
What do you think of `fetchAndStoreWebhook` ?

Reply:
Right!